### PR TITLE
Bug 1883790: Revert "force cert rotation every couple days for development: aggregator-client-signer"

### PR DESCRIPTION
**Pull request by sttts**
This reverts commit e1094f5fffd3f6e74d22d6c4b803e963e8dda171.

### CODE REVIEW

**Comment by sttts** [comment]
/hold

until master and release-4.7 are disconnected.

File: pkg/operator/certrotationcontroller/certrotationcontroller.go [hunk 1]
```diff
@@ -133,8 +133,8 @@ func newCertRotationController(
 		certrotation.SigningRotation{
 			Namespace:              operatorclient.OperatorNamespace,
 			Name:                   "aggregator-client-signer",
-			Validity:               30 * rotationDay,
-			Refresh:                15 * rotationDay,
+			Validity:               30 * defaultRotationDay,
```
Comment (p0lyn0mial) [inline pkg/operator/certrotationcontroller/certrotationcontroller.go:136]: what about other componenets ?

Comment (p0lyn0mial) [reply pkg/operator/certrotationcontroller/certrotationcontroller.go:136]: like `KubeAPIServerToKubeletClientCert` ?

Comment (sttts) [reply pkg/operator/certrotationcontroller/certrotationcontroller.go:136]: Good question. Have we changed it for other components? This is just a rebase of the same PR we had for the last couple of releases.

**Review by soltysh — APPROVED** [review]
/lgtm

**Pull request merged**